test/e2e/density.go

```diff
@@ -210,6 +214,7 @@ var _ = Describe("Density", func() {
 				PodStatusFile:        fileHndl,
 				Replicas:             totalPods,
 				MaxContainerFailures: &MaxContainerFailures,
+				Timeout:              time.Duration(math.Ceil(float64(itArg.podsPerNode)/minPodStartupThroughput)) * time.Minute,
```

Comment:
Having the throughput in pods/minute is very misleading (because everything else everywhere else is in pods/second).
Can you change it to 0.25 (so that we have per-second throughput)?